src/pages/Results/pages/Results/pages/FootO/pages/Splits/FootOSplits.tsx

```diff
@@ -158,7 +161,9 @@ export default function FootOSplits(
 
   const renderRadiosView = () => (
     <Box>
-      <ExperimentalFeatureAlert />
+      <Box sx={{ padding: "16px 16px 0 16px" }}>
+        <ExperimentalFeatureAlert />
```

Comment:
I think I missed some experimental place (radios appear on results screen I believe), maybe in another PR you can do it @sergiogpajares